x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/exceptions/exceptions_table.tsx

```diff
@@ -0,0 +1,195 @@
+  EuiLoadingContent,
+  EuiProgress,
+  EuiFieldSearch,
+} from '@elastic/eui';
+import styled from 'styled-components';
+import { History } from 'history';
+import { set } from 'lodash/fp';
+
+import { useKibana } from '../../../../../../common/lib/kibana';
+import { useExceptionLists } from '../../../../../../shared_imports';
+import { FormatUrl } from '../../../../../../common/components/link_to';
+import { HeaderSection } from '../../../../../../common/components/header_section';
+import { Loader } from '../../../../../../common/components/loader';
+import { Panel } from '../../../../../../common/components/panel';
+import * as i18n from './translations';
+import { AllRulesUtilityBar } from '../utility_bar';
+import { LastUpdatedAt } from '../../../../../../common/components/last_updated';
+import { AllExceptionListsColumns, getAllExceptionListsColumns } from './columns';
+import { useAllExceptionLists } from './use_all_exception_lists';
+
+// Known lost battle with Eui :(
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const MyEuiBasicTable = styled(EuiBasicTable as any)`` as any;
+
+export type Func = () => void;
+export interface ExceptionListFilter {
+  name?: string | null;
+  list_id?: string | null;
+  created_by?: string | null;
+}
+
+interface ExceptionListsTableProps {
+  history: History;
+  hasNoPermissions: boolean;
+  loading: boolean;
+  formatUrl: FormatUrl;
+}
+
+export const ExceptionListsTable = React.memo<ExceptionListsTableProps>(
+  ({ formatUrl, history, hasNoPermissions, loading }) => {
+    const {
+      services: { http, notifications },
+    } = useKibana();
+    const [filters, setFilters] = useState<ExceptionListFilter>({
+      name: null,
+      list_id: null,
+      created_by: null,
+    });
+    const [loadingExceptions, exceptions, pagination, refreshExceptions] = useExceptionLists({
+      errorMessage: i18n.ERROR_EXCEPTION_LISTS,
+      filterOptions: filters,
+      http,
+      namespaceTypes: ['single', 'agnostic'],
+      notifications,
+      showTrustedApps: false,
+    });
+    const [loadingTableInfo, data] = useAllExceptionLists({
+      exceptionLists: exceptions ?? [],
+    });
+    const [initLoading, setInitLoading] = useState(true);
+    const [lastUpdated, setLastUpdated] = useState(Date.now());
+
+    const handleDelete = useCallback((id: string) => () => {}, []);
+
+    const handleExport = useCallback((id: string) => () => {}, []);
+
+    const exceptionsColumns = useMemo((): AllExceptionListsColumns[] => {
+      return getAllExceptionListsColumns(handleExport, handleDelete, history, formatUrl);
+    }, [handleExport, handleDelete, history, formatUrl]);
+
+    const handleRefresh = useCallback((): void => {
+      if (refreshExceptions != null) {
+        setLastUpdated(Date.now());
+        refreshExceptions();
+      }
+    }, [refreshExceptions]);
+
+    useEffect(() => {
+      if (initLoading && !loading && !loadingExceptions && !loadingTableInfo) {
```

Comment:
It looks like `loadingTableInfo` is `true` when there are no exceptions in the system. This causes the Exceptions table to appear to load forever.
![image](https://user-images.githubusercontent.com/616158/102218124-85336600-3eab-11eb-9f10-943464569bea.png)

It looks like ultimately the problem is here, since we never actually make an API call and the default for `loading` is `true` in this hook.
https://github.com/elastic/kibana/pull/85465/files#diff-b0bc3e336feb76f1d84971f078e0b16884fe88ea14c719ce5b1ed82c4a9c4c26R100

Reply:
This was fixed in the code I didn't push from last night 🤦‍♀️ Pushed in!